Detections/AlsidForAD/ADAttacksPathways.yaml

```diff
@@ -0,0 +1,28 @@
+id: 9649e203-3cb7-47ff-89a9-42f2a5eefe31
+name: Alsid Active Directory attacks pathways
+description: |
+    'Searches for triggered Indicators of Exposures related to Active Directory attacks pathways'
+severity: Low
+requiredDataConnectors:
+  - connectorId: AlsidForAD
+    dataTypes:
+      - AlsidForADLog_CL
+queryFrequency: 2h
+queryPeriod: 2h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1110
+query: |
+  let SeverityTable=datatable(Severity:string,Level:int) [
+  "low", 1,
+  "medium", 2,
+  "high", 3,
+  "critical", 4
+  ];
+  afad_parser
+  | where MessageType == 0 and (Codename == "C-PRIV-ACCOUNTS-SPN" or Codename == "C-SDPROP-CONSISTENCY" or Codename == "C-DANG-PRIMGROUPID" or Codename == "C-GPO-HARDENING" or Codename == "C-DC-ACCESS-CONSISTENCY" or Codename == "C-DANGEROUS-TRUST-RELATIONSHIP" or Codename == "C-UNCONST-DELEG" or Codename == "C-ABNORMAL-ENTRIES-IN-SCHEMA")
```

Comment:
Missed this on the first pass.  Much more readable if you create a list for this and use the 'in' operator.  Change to -

let codeNameList = ("C-PRIV-ACCOUNTS-SPN", "C-SDPROP-CONSISTENCY", "C-DANG-PRIMGROUPID", "C-GPO-HARDENING", "C-DC-ACCESS-CONSISTENCY", "C-DANGEROUS-TRUST-RELATIONSHIP", "C-UNCONST-DELEG", "C-ABNORMAL-ENTRIES-IN-SCHEMA");
afad_parser
| where MessageType == 0 and codeNameList in~ (codeNameList)